ducktape/utils/util.py

```diff
@@ -53,3 +53,12 @@ def package_is_installed(package_name):
 def ducktape_version():
     """Return string representation of current ducktape version."""
     return __ducktape_version__
+
+def load_function(func_module_path):
+    """Loads and returns a function from a module path seperated by '.'s"""
+    module, function = func_module_path.rsplit(".", 1)
+    try:
+        return getattr(importlib.import_module(module), function)
+    except AttributeError:
+        raise Exception("Function could not be loaded from the module path {}, "
```

Comment:
```python
except AttributeError as e:
    raise Exception(...) from e
```

Reply:
doesn't look like this is compatable with python2 :/ 
```
try:
...     raise Exception('hi')
... except Exception as e:
...     raise Exception('there') from e
  File "<stdin>", line 4
    raise Exception('there') from e
                                ^
SyntaxError: invalid syntax

```

Reply:
_mumbles swear words into the beard_

Reply:
python2 man >:(